monitor/utils/message.js

```diff
@@ -66,10 +66,32 @@ const unclaimedHomeToForeignRequests = () => {
   }
 }
 
+const manuallyProcessedAMBHomeToForeignRequests = () => {
+  if (MONITOR_HOME_TO_FOREIGN_ALLOWANCE_LIST) {
+    const allowanceList = readAccessListFile(MONITOR_HOME_TO_FOREIGN_ALLOWANCE_LIST)
+    return e => {
+      const { sender, executor, decodedDataType } = normalizeAMBMessage(e)
+      return (
+        decodedDataType.manualLane ||
```

Comment:
Is this more logical to check this flag only if `(!allowanceList.includes(sender.toLowerCase()) && !allowanceList.includes(executor))` is false?

Reply:
I don't see the difference between these options. Why is it more logical?

Reply:
Because it reflects the logic which we put behind these code changes: for the senders/receivers included in the allowance list we check if the extension set the `manualLane` flag.

Even from performance point of view we can say that there could be bunch extensions included in the allowance list and only few of them in relatively small number of transactions will set the `manualLane` flag. 

Here is an illustration:
For example, we have 10 extensions deployed for AMB, we include in the allowance list only two of them. So, it means that we don't need to check the `manualLane` flag for 8 extensions at all but the suggested code will do this.

Reply:
Ok, now it makes some sense for me. Thanks.